src/commands/dnd/campaign/session/response.rs

```diff
@@ -37,3 +38,37 @@ pub async fn respond(
 
     responses::success(ctx, "Response recorded.").await
 }
+
+/// Responds to a D&D session (DM only), can respond for others
+#[poise::command(slash_command, check = "checks::dm_check")]
+pub async fn dmrespond(
+    ctx: Context<'_>,
+    #[description = "The ID of the session you're responding to"] session_id: i32,
+    #[description = "Who is going? Defaults to you."] respondee: Option<serenity::User>,
+    #[description = "Are they going?"] response: ResponseChoice,
+) -> Result<(), Error> {
+    if !session_ops::does_session_exist(ctx, session_id) {
+        return responses::failure(ctx, "Session not found.").await;
+    }
+
+    let response = match response {
+        ResponseChoice::Yes => 1,
+        ResponseChoice::No => 0,
+    };
+
+    let respondee_id = match respondee {
+        Some(respondee) => respondee.id,
+        None => ctx.author().id,
+    };
```

Comment:
```suggestion
```
Going with the suggestion above renders this useless.

Reply:
[5605995](https://github.com/dotunwrap/dm-helper/pull/26/commits/560599526cb7babef49d00c0ecd2e2bd420e9e8c)